launch_test.go

```diff
@@ -1059,6 +1059,53 @@ func TestFetchDefaultMeta(t *testing.T) {
 	}
 }
 
+func TestFetchBuildParameterMeta(t *testing.T) {
+	initCoverageMeta()
+	oldWriteFile := writeFile
+	defer func() { writeFile = oldWriteFile }()
+	var defaultMeta []byte
+	mockMeta := make(map[string]interface{})
+	mockMeta["foo"] = "bar"
+	mockMeta["meta"] = map[string]interface{}{
+		"baz":     "qux",
+		"summary": map[string]string{"comment": "it should be deleted"},
+	}
```

Comment:
Are these meta needed for this test case? I think It should focus on what to check.

Reply:
I deleted it because it is an unnecessary setting for the test.